### PR TITLE
enable release plugin

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Comment by patriknw** [comment]
I missed on thing here, will update

File: build.sbt [hunk 1]
```diff
@@ -28,13 +28,13 @@ val coreDependencies = Seq(
 )
 
 val commonSettings =
-sonatypeSettings ++ scalariformSettings ++ Seq(
+  scalariformSettings ++ Seq(
   version := "0.11.0-SNAPSHOT",
-  organization := "com.softwaremill.reactivekafka",
+  organization := "com.typesafe.akka",
+  organizationName := "Lightbend",
```
Comment (ktoso) [inline build.sbt]: There's organisation in the maven metadata: `       <connection>scm:git:git@github.com:softwaremill/reactive-kafka.git</connection>`

File: project/Publish.scala [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  def akkaPomExtra = {
+    <scm>
+      <url>git@github.com:akka/reactive-kafka.git</url>
+      <connection>scm:git:git@github.com:akka/reactive-kafka.git</connection>
+    </scm>
+    <developers>
+      <developer>
+        <id>contributors</id>
+        <name>Contributors</name>
+        <email>akka-dev@googlegroups.com</email>
```
Comment (ktoso) [inline project/Publish.scala]: that mailing list is dead though hm. IMO OK to keep Krzysiek here.

File: build.sbt (hunk 1 above)
Comment (patriknw) [inline build.sbt]: yes, that was the missing piece, I forgot to remove the old publish settings

**Comment by kciesielski** [comment]
LGTM

**Pull request merged**